flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -597,8 +597,7 @@ protected void processInput(MailboxDefaultAction.Controller controller) throws E
             resumeFuture = inputProcessor.getAvailableFuture();
         } else if (changelogWriterAvailabilityProvider != null) {
             // currently, waiting for changelog availability is reported as busy
-            // todo: add new metric (FLINK-24402)
-            timer = null;
+            timer = new GaugePeriodTimer(ioMetrics.getIoBackPressuredTimePerSecond());
```

Comment:
The comment above is no longer true with this change, right?
When a task is waiting for changelog availability (but not blocked in writer), it will be reported as idle.

Won't such a change without adjusting the UI be confusing to the users?

cc: @pnowojski

Reply:
> When a task is waiting for changelog availability (but not blocked in writer), it will be reported as idle.

Are you sure? With the code in this PR as it is right now, I think task blocked on this changelog availability future, would be reported as back pressured.

> Won't such a change without adjusting the UI be confusing to the users?

As far as I can tell, this PR will move "waiting for changelog availability" from "busy" to "backpressured" in the UI. From this perspective, this should be fine in the UI (UI only displays `busy/backpressured/idle` and `backpressured = soft + hard`) % we are changing semantic pf those metric a bit.

Reply:
Yes, sorry, that's what I wanted to say: "backpressured", not "idle".

So should the documentation be updated then?
E.g. in https://nightlies.apache.org/flink/flink-docs-master/docs/ops/monitoring/back_pressure/

> Internally, back pressure is judged based on the availability of output buffers. If a task has no available output buffers, then that task is considered back pressured. Idleness, on the other hand, is determined by whether or not there is input available.

(and the highlighted comment)

Reply:
Thanks for the review and analysis, I moved the `changelogBusyTime` into the busy time.

> So should the documentation be updated then?

Because we still treat `changelogBusyTime` as busy time, the documentation can stay as it is.